### ISSUE TITLE
PUGXMultiUserBundle Too few arguments to function FOS\UserBundle\Controller\RegistrationController::__construct()

### ISSUE THREAD

**Issue by antoine-salamitou** [PUGX/PUGXMultiUserBundle#156]
Hi i am trying to use PUGXMultiUserBundle on Symfony 3.4, i followed the documentation step by step and when i am trying to go on 
`http://localhost:8888/web/app_dev.php/register/user-one`
i have this error : 
Type error: Too few arguments to function FOS\UserBundle\Controller\RegistrationController::__construct(), 0 passed in /Users/Antoine/Desktop/pugx/Symfony/var/cache/dev/ContainerSmtb8gw/getPugxMultiUser_RegistrationManagerService.php on line 8 and exactly 4 expected

i have tried everything !

**Comment by garak** [comment]
I don't think this bundle is suitable for Symfony 3.4 or for any other recent version of Symfony

**Comment by antoine-salamitou** [comment]
ok thx too bad

**Comment by Tomasz-Kolasa** [comment]
I've wasted 4 hours configuring the PUGXMultiUserBundle and it turns out it's not suitable for Symfony 3.4 - what a waste of time!
Anyway, does anyone could tell me if theres's any alternative way to use multiple users entities in Symfony?

**Comment by garak** [comment]
@Tomasz-Kolasa of course there's a way: just do not use FOSUserBundle
